src/buildCommon.js

```diff
@@ -63,6 +57,65 @@ var mathrm = function(value, mode, color, classes) {
 };
 
 /**
+ * Makes a symbol in the default font for mathords and textords.
+ */
+var mathDefault = function(value, mode, color, classes, type) {
+    if (type === "mathord") {
+        return mathit(value, mode, color, classes);
+    } else if (type === "textord") {
+        return makeSymbol(
+            value, "Main-Regular", mode, color, classes.concat(["mathrm"]));
+    } else {
+        throw new Error("unexpected type: " + type + " in mathDefault");
+    }
+};
+
+/**
+ * Makes a symbol in the italic math font.
+ */
+var mathit = function(value, mode, color, classes) {
+    if (/[0-9]/.test(value.charAt(0)) ||
+        utils.contains(["\u0131", "\u0237"], value) ||
```

Comment:
Also can you indent the tests in the if statement an extra indent so it doesn't line up with the body?

```
if (test &&
        test) {
    body;
}
```